en/components/grids_templates/row_editing.md

```diff
@@ -299,6 +299,199 @@ If you want the buttons to be part of the keyboard navigation, then each on of t
 </ng-template>
  ```
 
+### Styling
+
+Using the [IgniteUI theme engine](../themes/index.md), we can greatly alter the row-editing overlay. 
+In the below steps, we'll go through the process of changing the look of our row-editing banner by passing custom templates for with the `igxRowEditActions` and `igxRowEditText` directives and making use of the [`igx-grid-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-theme), [`igx-banner-theme`]({environment:sassApiUrl}/index.html#function-igx-banner-theme), [`igx-input-group-theme`]({environment:sassApiUrl}/index.html#function-igx-input-group-theme) and [`igx-button-theme`]({environment:sassApiUrl}/index.html#function-igx-button-theme).
```

Comment:
Diff doesn't do well with this long line, so:
- "In the steps below, ..:"
- again, no dash on Row Editing.

You should probably explain where the banner came from all of a sudden(no mention of it so far) and that the overlay makes use of the banner layout and uses buttons, thus their themes apply.

Also not sure what the deal is with cell styling, that should be in the normal editing topic, you can link to that. Meaning, drop the grid and input group themes.